docs/docs/behaviors/bluetooth.md

```diff
@@ -32,6 +32,7 @@ Here is a table describing the command for each define:
 | `BT_NXT` | Switch to the next profile, cycling through to the first one when the end is reached.                                                                     |
 | `BT_PRV` | Switch to the previous profile, cycling through to the last one when the beginning is reached.                                                            |
 | `BT_SEL` | Select the 0-indexed profile by number. Please note: this definition must include a number as an argument in the keymap to work correctly. eg. `BT_SEL 0` |
+| `BT_DIS` | Disconnects all bluetooth profiles. This command disables bluetooth advertising until a board reset is issued.                                            |
```

Comment:
I'm not sure I love the "reset if you want your keyboard to work again" aspect of this. What is the use case here?

If the target is "make it not wake up my laptop when it's in my bag", I'd rather we solve that with soft lock or sleep logic, not just some BLE change like this.

Reply:
In addition to not registering keypresses, it may save energy. But I am not sure about that. You could also disable bluetooth completely but continue using it over USB with this functionality.